bindata/manifests/switchdev-config/files/switchdev-configuration-after-NM.sh.yaml

```diff
@@ -0,0 +1,35 @@
+mode: 0755
+overwrite: true
+path: "/usr/local/bin/switchdev-configuration-after-NM.sh"
+contents:
+  inline: |
+    #!/bin/bash
+    set -eux
+    input="/etc/sriov_config.json"
+
+    if [ ! -f $input ]; then
+      echo "File /etc/sriov_config.json not exist."
+      exit
+    fi
+
+    # Required for NetworkManager configuration(e.g. bond) to settle down
+    sleep 3
```

Comment:
what happens if we dont have this sleep ? do you know what configurations need to settle down ?

Reply:
What happened was that VF LAG didn't take effect (there is the dmesg indicating the error).

Reply:
@adrianchiris  any follow-up questions ?

Reply:
no additional questions. VF Lag has some quirks :(